docs/source/getting_started.rst

```diff
@@ -0,0 +1,39 @@
+Getting Started
+===============
+
+The Data Resource Generator (DRG) is a database and API as configuration application. It is primarily used as a core component of BrightHive Data Trusts but has many :ref:`use-cases<use-cases>`. 
+
+Definitions
+-----------
+
+Data Trust
+    Legal, technical, and governance frameworks that enable networks of organizations to securely and responsibly share and collaborate with data, generating new insights and increasing their combined impact.
+
+Data Resource
+    Loosely defined as data that is owned by or stewarded over by members of Data Trusts. These are core elements of a Data Trust. In the context of the DRG, a Data Resource is a relational database with a RESTful API. 
+
+From the technical perspective, a BrightHive Data Resource is an entity comprised of the following elements:
```

Comment:
These terms extend the definition of "Data Resource." Is there a way to make the nesting more clear? Otherwise, your reader's eyes might skip over your framing sentence ("a BrightHive Data Resource is an entity comprised of...").